core/templates/pages/maintenance-page/maintenance-page.mainpage.html

```diff
@@ -16,7 +16,7 @@
             <nav class="navbar navbar-default oppia-navbar oppia-prevent-selection" role="navigation">
               <div class="navbar-container">
                 <div class="navbar-header protractor-test-navbar-header float-left">
-                  <a class="oppia-navbar-brand-name oppia-transition-200" href="/">
+                  <a class="oppia-navbar-brand-name oppia-transition-200" href="/splash">
```

Comment:
Same question about the reintroduction of /splash.

Reply:
PTAL in the thread [above](https://github.com/oppia/oppia/pull/9109#discussion_r410668909).